service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -600,6 +614,73 @@ void shouldNotifyAdminOfPlacementApplicationUpload() throws Exception {
         );
     }
 
+    @Test
+    void shouldSendNotificationToPartiesWhenAddedToCaseByEmail() throws IOException, NotificationClientException {
+        final Map<String, Object> expectedParameters = getPartyAddedByEmailNotificationParameters();
+
+        List<Representative> representatives = new ArrayList<>();
+        Representative representative = Representative.builder()
+                .email("joe-blogs@gmail.com")
+                .servingPreferences(EMAIL).build();
+        representatives.add(representative);
```

Comment:
We could extract this and just have a private method that returns List<Representative>. Passing in `servingPref` as an arg. Used in all tests by the looks of it.

Reply:
done :)